config/defaults.go

```diff
@@ -0,0 +1,308 @@
+package config
+
+import (
+	"github.com/spf13/viper"
+)
+
+// DefaultConfigFilePath is in the root folder by default
+const DefaultConfigFilePath = "config.json"
+
+// General defaults
+const (
```

Comment:
From my point of view all these consts are redundant and add unnecessary boilerplate code

Reply:
Why? :(

These constants are used both in viper and in flags, that's why we need them. Plus it's better to see / use / set them like that, instead of looking into viper configuration and/or flags configuration. But that's just my opinion, WDYT?

Reply:
Removed :)